packages/devtools/lib/src/debugger/debugger.dart

```diff
@@ -612,8 +618,8 @@ class DebuggerScreen extends Screen {
       ..flex()
       ..layoutVertical();
 
-    debuggerState.onBreakpointsChanged.listen((List<Breakpoint> breakpoints) {
-      breakpointsView.showBreakpoints(breakpoints);
+    debuggerState.breakpoints.addListener(() {
```

Comment:
add back in the listen on assignment and add a TODO to figure out if it is really needed. @devoncarew might have a better idea.

Reply:
Added with TODO pointing to the existing issue.